src/modules/nats/nats_mod.c

```diff
@@ -519,6 +614,7 @@ int init_nats_server_url_add(char *url)
 		n = n->next;
 	}
 	n = _init_nats_server_list_new(url);
+	_init_nats_server_conn(n);
```

Comment:
this initiates connections to the servers but each subscriber still maintains a separate connection to each server. i think it would make sense to change all subscriber workers to use the same connections as publishers